src/core_plugins/kibana/public/visualize/index.js

```diff
@@ -23,6 +23,14 @@ import { VisualizeConstants } from './visualize_constants';
 import savedObjectRegistry from 'ui/saved_objects/saved_object_registry';
 import savedVisusalizationProvider from 'plugins/kibana/visualize/saved_visualizations/saved_visualization_register';
 
+import { CreateOrDeleteButton } from './listing/create_or_delete_button';
+
+import uiModules from 'ui/modules';
```

Comment:
Question: For app specific react components, should we add the connection at the base of the app (e.g. index.js) or where they are used (here).  I'm not sure which I like better.